assets/styles/main.css

```diff
@@ -192,16 +192,25 @@ p {
   }
 
   .testimonials {
-    display: flex;
-    justify-content: space-between;
+    display: inline-grid;
+    grid-template-columns: repeat(3, 1fr);
+    grid-template-rows: repeat(10, 1fr);
+    gap: 1rem;
   }
 
-  .testimonial-card {
-    flex-basis: 32%;
-    margin-bottom: 0;
+  .testimonial-card:nth-child(1) {
+    grid-column: 1 / 2;
+    grid-row: 1 / 9;
   }
 
-  .testimonial-card:nth-child(1) {
-    align-self: center;
+  .testimonial-card:nth-child(2) {
+    grid-column: 2 / 3;
+    grid-row: 2 / 10;
+  }
+
+  .testimonial-card:nth-child(3) {
+    grid-column: 3 / 4;
+    grid-row: 3 / 11;
   }
+
 }
```

Comment:
why u didnt use flexbox, I think with grid this adds a lot of code but thank u for ur contrib